### PR TITLE
amodv2: create Message Length Trigger

### PR DESCRIPTION
**Pull request by Pedro-Pessoa**
This adds 2 new triggers to amodv2:
1. Message with more than x characters
2. Message with less than x characters

Character count is done via `utf8.RuneCountInString`

### CODE REVIEW

File: automod/triggers.go [hunk 1]
```diff
@@ -1332,3 +1333,66 @@ func (mat *MessageAttachmentTrigger) CheckMessage(triggerCtx *TriggerContext, cs
...
+
+func (ml *MessageLengthTrigger) CheckMessage(triggerCtx *TriggerContext, cs *dstate.ChannelState, m *discordgo.Message, mdStripped string) (bool, error) {
+	dataCast := triggerCtx.Data.(*MessageLengthTriggerData)
+
+	if ml.Inverted {
+		if utf8.RuneCountInString(m.Content) < dataCast.Length {
+			return true, nil
+		}
+
+		return false, nil
```
Comment (jo3-l) [inline automod/triggers.go:1390]:
```suggestion
		return utf8.RuneCountInString(m.Content) < dataCast.Length, nil
```

File: automod/triggers.go [hunk 2]
```diff
@@ -1332,3 +1333,66 @@ func (mat *MessageAttachmentTrigger) CheckMessage(triggerCtx *TriggerContext, cs
...
+	}
+
+	if utf8.RuneCountInString(m.Content) > dataCast.Length {
+		return true, nil
+	}
+
+	return false, nil
```
Comment (jo3-l) [inline automod/triggers.go:1397]:
```suggestion
	return utf8.RuneCountInString(m.Content) > dataCast.Length, nil
```

**Review by jo3-l — COMMENTED** [review]
think the logic can be simplified to this

**Comment by ashishjh-bst** [comment]
@Pedro-Pessoa  branch needs rebasing to dev to resolve conflicts

**Comment by Pedro-Pessoa** [comment]
Should be good now

**Pull request merged**